src/command_modules/azure-cli-role/azure/cli/command_modules/role/tests/role_scenario_test.py

```diff
@@ -0,0 +1,26 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+import uuid
+import mock
+from azure.cli.testsdk import ScenarioTest
+
+
+class RoleScenarioTest(ScenarioTest):
+
+    def enable_large_payload(self, size=8192):
+        from azure_devtools.scenario_tests import LargeResponseBodyProcessor
```

Comment:
We should consider moving this into the test base. It is used in many places to work around what I would consider a bug in devtools.

Reply:
we can port https://github.com/Azure/azure-cli/issues/4127 over